src/view-port-cache.js

```diff
@@ -0,0 +1,26 @@
+import {transient} from 'aurelia-dependency-injection';
+
+@transient()
+export class ViewPortCache {
+  controllers = {};
```

Comment:
I remember we had to ensure correct cache lookup by
`controllers = Object.create(null);`
I guess it's should be applied here also..